app/src/main/java/comp3350/mealbuddy/objects/Food.java

```diff
@@ -0,0 +1,49 @@
+package comp3350.mealbuddy.objects;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.EnumMap;
+import java.util.Enumeration;
+import java.util.Hashtable;
+
+public class Food {
+
+
+    private String name;
+
+    private ArrayList<String>labels;
+
+    private EnumMap macroDictionary;
+
+    private EnumMap microDictionary;
+
```

Comment:
a lot of wasted space.
Also, we shouldn't be using an EnumMap here, a Food has a list of nutrients, it doesn't contain ALL possible nutrients. example code below:
```suggestion
    private String name;
    private ArrayList<String>labels;
    private ArrayList<Nutrient>nutrients;
```